docs/tutorial_custom.rst

```diff
@@ -90,9 +120,11 @@ only when using the DataFrame.
 
 .. code:: python
 
-   def custom_func_add(inst, optional_param1=False, optional_param2=False):
-       return {'data': 2.*inst['mlt'], 'name': 'double_mlt',
-               'long_name': 'doubledouble', 'units': 'hours'}
+   def custom_func_add(inst, param1, optional_param1=False,
```

Comment:
Recommend changing function input to be useful.  Like `factor=1` and `param_name='mlt'` and then using these in the dict below.